services/storage/src/simcore_service_storage/models.py

```diff
@@ -353,7 +353,12 @@ def none(cls) -> "AccessRights":
 
 class PathMetaData(BaseModel):
     path: Path
-    display_path: Path
+    display_path: Annotated[
+        Path,
+        Field(
+            description="Path with names instead of IDs (URL Encoded by parts as names may contain '/')"
```

Comment:
I wonder if we should switch to something different to / as separators and avoid all this encoding hell :-)

Reply:
well since we have no restrictions on project names and node names, not sure I see it.